.github/workflows/ci.yml

```diff
@@ -617,7 +617,9 @@ jobs:
 
       - name: Get version
         id: version
-        run: echo "version=${GITHUB_REF##*/}" | tee --append "$GITHUB_OUTPUT"
+        run: |
+          echo "version=${GITHUB_REF##*/}" | tee --append "$GITHUB_OUTPUT"
+          echo "protocol_version=$(curl https://raw.githubusercontent.com/sass/sass/HEAD/spec/EMBEDDED_PROTOCOL_VERSION)" | tee --append "$GITHUB_OUTPUT"
```

Comment:
There's a risk that this will run into quota limits. We should either pass the GitHub token to it, or find a pre-existing action we can use to download a file with token support. (We could also just use actions/checkout but that'll take longer than necessary.)